.github/workflows/maven.yml

```diff
@@ -0,0 +1,68 @@
+name: Arquillian-Cube Maven Build
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - '**'
```

Comment:
Have we considered skipping the checks if the PR is just about e.g.: docs? See for instance a WIP [here](https://github.com/Intersmash/intersmash/pull/66/files#diff-d978a3dadf11b38be2ec45facd9d1f5a3785546602451eeb09c2c3cf20a104aaR9)

Reply:
done